### DISCUSSION TITLE
Is there a "proper" place to store my TS libraries?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#11418] [General - Components / Directives / etc] [answered]
I have several TypeScript libraries (specifically written for the app, it is basically a centralized repository for several exported functions I do not want to repeat in different `<script>`)

Right now I put them in `components` and import via `import {something} form 'components/muylibfile.ts'`

It does not look right, though, because this is not a component.

I thought about creating a `lib` directory and import `src/lib/...` but there may ab a "proper" place to store these libraries in Quasar.

**Comment by metalsadman** [comment, marked as answer]
we don't have a guideline for that, put it in a folder where you think it makes sense or suited or convenient etc... you can also make your own aliased folder https://quasar.dev/quasar-cli/handling-webpack#adding-webpack-aliases.
